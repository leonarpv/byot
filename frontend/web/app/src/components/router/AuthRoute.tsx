import * as React from 'react';
import {AuthRouteBase} from '@byot-frontend/web-common/src/components/router/AuthRouteBase';
import {Router} from '../../router/Router';
import {RouteProps} from 'react-router-dom';

interface Props extends RouteProps {}

export const AuthRoute: React.FC<Props> = (props: Props) => {
  return <AuthRouteBase authUrl={Router.login.URI()} {...props} />;
};
