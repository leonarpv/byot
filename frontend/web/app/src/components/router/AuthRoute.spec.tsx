import * as React from 'react';
import {render} from '@testing-library/react';
import {AuthRoute} from './AuthRoute';

describe('<AuthRoute/>', () => {
  it('should render', () => {
    const {container} = render(<AuthRoute />);
    expect(container).toMatchSnapshot();
  });
});
