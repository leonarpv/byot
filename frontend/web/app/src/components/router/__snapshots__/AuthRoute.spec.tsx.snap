// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthRoute/> should render 1`] = `
<div>
  <div
    data-testid="auth-route-base"
  />
</div>
`;
